lms/envs/test.py

```diff
@@ -474,9 +474,13 @@
 FEATURES['CERTIFICATES_HTML_VIEW'] = True
 
 # Set dummy values for profile image settings.
-PROFILE_IMAGE_BACKEND = 'django.core.files.storage.FileSystemStorage'
-PROFILE_IMAGE_DOMAIN = 'http://example-storage.com/'
-PROFILE_IMAGE_URL_PATH = 'profile_images/'
+PROFILE_IMAGE_BACKEND_CONFIG = {
+    'storage_class': 'storages.backends.overwrite.OverwriteStorage',
+    'storage_kwargs': {
+        'location': TEST_ROOT / 'uploads/profile_images/',
```

Comment:
Good point. I think I'll just make it point to `MEDIA_ROOT` (`TEST_ROOT + 'uploads'` in tests), since that does exist, and I can't think of any reason why we should enforce a particular directory structure for the test environment.